extensions/wikia/VideoHandlers/VideoHandlers.setup.php

```diff
@@ -278,7 +274,6 @@
 	27 => 'Twitchtv',
 	28 => 'Ooyala',
 	29 => 'Iva',
-	30 => 'Snappytv',
```

Comment:
I created the ticker for removing not used code: https://wikia-inc.atlassian.net/browse/CONCF-159.
For now I'm not going to fix the indexing in array as it's not needed at all.